### ISSUE TITLE
Decade setting doesn’t effect weapons

### ISSUE THREAD

**Issue by PSYKO-nz** [akaAgar/briefing-room-for-dcs#92]
We like to fly with limitations on our weapons so that we don’t just do mission after mission with jdams and JSOWs and the like where all you gotta do is plug in coordinates and get in rang then pickle

we have tried a few times to change the decade hoping that it would change the available weapons however there has been no change. So I’m assuming this has not been implemented yet.

at the moment we set the decade, let the mission builder pick the year and then google the weapons we want to use to see if they were available. 

Would be epic if the mission editor decade setting did this for us!

Again we love briefing room, these are all just things we noticed during play that we thought we should bring to your attention.

you guys are just awesome for making this

**Comment by john681611** [comment]
Thanks, guys its low on our priority but you got my mental gears turning. I'll see if I can implement the base functionality for initial aircraft loadouts but in terms of actual load-outs for each aircraft, I think we will need to lean on the community to implement it into every aircraft.

Airbase limitations are another thing entirely.

**Comment by john681611** [comment]
Base functionality has been implemented. We are asking interested parties to help generate the data.

**Comment by PSYKO-nz** [comment]
Awesome, I put a message in the discussions tab the other day about bet testing for you, here’s the message so I don’t have to type it again lol...

“As ive said I’m rubbish at coding however I do have a crew of 9 (closed group, locked server, so no chances of the file getting out) who use this mission builder every Wednesday to generate missions on our dcs night.

if it is possible or needed we are happy to beta test for you. Happy to run through anything that needs testing in the wild for you,

then we can report bugs and ideas for development

if your interested please let me know.

All we would need is a beta version of the installer. Or a way to update the current one”

In other words. If you need feedback we are ready and willing

**Comment by john681611** [comment]
I wrote a bit about configuring loadouts its probably a bit too technical but it could allow you to help us with loadouts:
https://github.com/akaAgar/briefing-room-for-dcs/blob/main/Modder's%20manual.md#aircraft

Basically, you can open the .miz file with 7zip and use F3 to look at the mission file 
Then search for `["pylons"] = ` and that should have the weapon loadout of that aircraft. As I say in the docs best just spawn 1 aircraft and save the mission so you don't end up with multipul loadouts. We don't have a beta build as per you can clone to code down and build the project and that gives you a working version of the latest code.

**Comment by PSYKO-nz** [comment]
Do you have a quick how to on that?

**Comment by john681611** [comment]
Tbh I need to make some proper data extraction docs (unless we can find a better way than via missions).
1. Open DCS -> editor -> place jet with loadout wanted (one jet makes it easier) -> save the mission
1. Open MIssion File with 7.Zip
1. View `mission`  (F3) 
1. Search `pylons`
1. Copy code that looks like 
```
 [1] = {["CLSID"] = "{5CE2FF2A-645A-4197-B48D-8720AC69394F}" },
 [2] = {["CLSID"] = "LAU_117_AGM_65F" },
 [3] = {["CLSID"] = "{BRU33_2X_GBU-12}" },
 [4] = {["CLSID"] = "{40EF17B7-F508-45de-8566-6FFECC0C1AB8}" },
 [5] = {["CLSID"] = "{FPU_8A_FUEL_TANK}" },
 [6] = {["CLSID"] = "{40EF17B7-F508-45de-8566-6FFECC0C1AB8}" },
 [7] = {["CLSID"] = "{BRU33_2X_GBU-12}" },
 [8] = {["CLSID"] = "{F16A4DE0-116C-4A71-97F0-2CF85B0313EC}" },
 [9] = {["CLSID"] = "{5CE2FF2A-645A-4197-B48D-8720AC69394F}" },
 ```
 
1. Insert into Jet.Ini file in Database/Units like so
```
Payload.Decade2000.Task.Default.Pylon01={5CE2FF2A-645A-4197-B48D-8720AC69394F}
Payload.Decade2000.Task.Default.Pylon02=LAU_117_AGM_65F
Payload.Decade2000.Task.Default.Pylon03={BRU33_2X_GBU-12}
Payload.Decade2000.Task.Default.Pylon04={40EF17B7-F508-45de-8566-6FFECC0C1AB8}
Payload.Decade2000.Task.Default.Pylon05={FPU_8A_FUEL_TANK}
Payload.Decade2000.Task.Default.Pylon06={40EF17B7-F508-45de-8566-6FFECC0C1AB8}
Payload.Decade2000.Task.Default.Pylon07={BRU33_2X_GBU-12}
Payload.Decade2000.Task.Default.Pylon08={F16A4DE0-116C-4A71-97F0-2CF85B0313EC}
Payload.Decade2000.Task.Default.Pylon09={5CE2FF2A-645A-4197-B48D-8720AC69394F}
```

Generally Checkout the hornet config as that is most complete.

**Comment by PSYKO-nz** [comment]
Awesome. Thank you

now how about this bit?
. We don't have a beta build as per you can clone to code down and build the project and that gives you a working version of the latest code.

**Comment by john681611** [comment]
Yeah the main branch in this repo is the latest code. So if you can get a C#/.Net app working then you can run latest code
On the other hand I'm considering setting up some code to build a beta version each time we update the code.

**Comment by PSYKO-nz** [comment]
That may be a better easier idea, make it way easier for testing

**Comment by john681611** [comment]
Decade based weapons have been added it's now a case of filling out the data. We are still looking for someone to help with this
